interceptors/selector/doc.go

```diff
@@ -0,0 +1,12 @@
+/*
+Package selector
+
+`selector` a generic server-side selector middleware for gRPC.
+
+# Server Side Selector Middleware
+It allows to set check rules to whitelist or blacklist middleware such as Auth
```

Comment:
Thanks for the updates, do you mind changing these phrases to `allowlist` and `blocklist` consistently through this PR?
```suggestion
It allows to set check rules to allowlist or blocklist middleware such as Auth
```